include/sensors/core/mmr920C04.hpp

```diff
@@ -0,0 +1,281 @@
+#pragma once
+
+#include <optional>
+
+#include "can/core/ids.hpp"
+#include "can/core/messages.hpp"
+#include "common/core/logging.h"
+#include "sensors/core/mmr920C04_registers.hpp"
+#include "sensors/core/sensors.hpp"
+
+namespace mmr920C04 {
+
+using namespace can_ids;
+using namespace mmr920C04_registers;
+using namespace sensors_registers;
+
+template <class I2CQueueWriter, message_writer_task::TaskClient CanClient>
+class MMR92C04 {
```

Comment:
Looks like we do this in other instances, so if I'm alone in thinking that naming classes and namespaces solely after a part is alienating just ignore me.

Reply:
Yeah generally we name the "drivers" the part number, but I can add an additional comment in this file to say that this is a pressure driver class.